uktrade/data-hub-frontend
# Project summary filters

Superseded by #3984 

## Description of change

Makes the investment project summary component stay consistent with the filters picked in the listing component by using the new "summary" data from the api endpoint.

## Test instructions

With the personalised dashboard user feature flag enabled, go to the dashboard and select filters for investment projects - the chart should update to reflect the selected filters. It is also worth checking what happens when you navigate to the companies list tab and back again.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/1234577/132517960-ffec252c-ada1-4a85-90e1-8c4cb6a5aec1.png)

![image](https://user-images.githubusercontent.com/1234577/132518006-2b48641a-f92f-41a0-8b20-e9c9f85b58fe.png)

### After

![image](https://user-images.githubusercontent.com/1234577/133051959-5dba14d3-eb3b-4db8-9dac-d405b06b84a3.png)

![image](https://user-images.githubusercontent.com/1234577/133052079-58cf1bc6-e463-4040-ab65-d273d3fa7dc2.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
